app/views/reports/regions/details.html.erb

```diff
@@ -307,33 +308,41 @@
               </th>
             <% else %>
               <th colspan="6">
-                New registrations<sup>2</sup>
+                New registrations<sup>3</sup>
               </th>
               <th colspan="6">
-                Follow-up patients<sup>3</sup>
+                Follow-up patients<sup>4</sup>
               </th>
             <% end %>
           </tr>
           <tr class="sorts" data-sort-method="thead">
             <th class="row-label sort-label sort-label-small ta-center" data-sort-default>
               Facilities
             </th>
+            <th class="row-label sort-label sort-label-small ta-center" data-sort-method="number">
+              <% if current_admin.feature_enabled?(:report_with_exclusions) %>
+                Total assigned<br>
```

Comment:
Is the `br` necessary? It seems awkward to include it at this particular location. I'd rather have the word-wrapping do its thing if possible.

Reply:
I was just imitating line 332 below and trying to match the mocks. I'll agree that it isn't necessary and probably just makes our styling and css more finicky. I'll change it and line 332.